zimui/src/components/TopicHome.vue

```diff
@@ -88,6 +93,11 @@ const goToPreviousPage = () => {
 </script>
 
 <template>
+<div>
+    <div v-if="errMessage" class="error-message" >
```

Comment:
Looks like there is an indentation issue here

Reply:
I don't exactly understand what issue are you getting, I am encountering an error.